### PR TITLE
try bazel --batch on the canary job

### PR DESCRIPTION
**Pull request by BenTheElder**
@ixdy suggested this as an option for potentially mitigating the issues we've had with migrating to 0.7 so let's test this on the canary job.

### CODE REVIEW

**Comment by BenTheElder** [comment]
/area bazel
/area jobs
cc @ixdy

File: scenarios/kubernetes_bazel.py [hunk 1]
```diff
@@ -71,7 +85,7 @@ def get_changed(base, pull):
         '--noshow_progress',
         'set(%s)' % diff).split('\n')
 
-def query(kind, selected_pkgs, changed_pkgs):
+def query(args, kind, selected_pkgs, changed_pkgs):
     """
```
Comment (ixdy) [inline scenarios/kubernetes_bazel.py]: it is necessary to pass `args` through everywhere? should we just be passing `batch`, or something else more clever?

Comment (BenTheElder) [reply scenarios/kubernetes_bazel.py]: It's not necessary but we usually do this in the scenarios. I'll try to make this cleaner.

Comment (BenTheElder) [reply scenarios/kubernetes_bazel.py]: I wrapped up query/check/check_output for bazel in a class so we don't have to keep explicitly passing batch / args all over.

**Pull request merged**